src/zeit/content/volume/volume.py

```diff
@@ -103,27 +108,42 @@ def previous(self):
     def next(self):
         return self._find_in_order(self.date_digital_published, None, 'asc')
 
-    @property
-    def _all_products(self):
-        return [self.product] + self.product.dependent_products
-
     def _find_in_order(self, start, end, sort):
         if len(filter(None, [start, end])) != 1:
             return None
-        elastic = zope.component.getUtility(
-            zeit.retresco.interfaces.IElasticsearch)
-        result = elastic.search({'query': {'bool': {'filter': [
+        # Since `sort` is passed in accordingly, and we exclude ourselves,
```

Comment:
Wozu musste das umgebaut werden?

Reply:
Nur ein Refactoring und Umzug von Code aus dem script in z.c.volume